packages/google-analytics/src/components/index.tsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable react/no-danger */
+import React from "react";
+import { connect } from "frontity";
+import { Connect } from "frontity/types";
+import GoogleAnalytics from "../../types";
+import { getTrackerName } from "..";
+
+const Root: React.FC<Connect<GoogleAnalytics>> = ({ state }) => {
+  // Get Tracking ids from state.
+  const { trackingIds, trackingId } = state.ga;
+  const ids = trackingIds || (trackingId && [trackingId]) || [];
+
+  // Render Tracker code.
+  return (
+    <script
```

Comment:
It seems that this script should go in the `<head>` section:
> Copy and paste the global site tag to each web page you want to measure. Add the tag right after the opening <head> tag on each page.

https://support.google.com/analytics/answer/1008080?hl=en

Reply:
Good catch 👍